format/Message.fbs

```diff
@@ -78,6 +214,34 @@ table DictionaryBatch {
   isDelta: bool = false;
 }
 
+table Xxhash64 {
+}
+
+table ExtensionDigest {
```

Comment:
What's this?

Reply:
Left over from a different option for extensions, will remove.